### PR TITLE
Fixing keyboard does not activate on inputs #953

### PR DESCRIPTION
**Pull request by LuuOW**
## PR Checklist

Please check all that apply to this PR using "x":

- [ X ] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ X ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #953

## What is the current behavior?

The app doesn't show keyboard when pressing the input text box in the wallet to put the address or the amount to donate.

## What is the new behavior?

Now  the keyboar is properly popping up.

### CODE REVIEW

File: app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java [hunk 1]
```diff
@@ -397,19 +397,6 @@ public void onClick(View view) {
...
-                inputSendAddress.setHint(hasFocus ? getString(R.string.recipient_address_placeholder) : "");
-            }
-        });
-        inputSendAmount.setOnFocusChangeListener(new View.OnFocusChangeListener() {
-            @Override
-            public void onFocusChange(View view, boolean hasFocus) {
-                inputSendAmount.setHint(hasFocus ? getString(R.string.zero) : "");
-                inlineBalanceContainer.setVisibility(hasFocus ? View.VISIBLE : View.INVISIBLE);
-            }
-        });
```
Comment (akinwale) [inline app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java:412]: Why were these lines removed? They perform bits of functionality when the input views gain focus.

**Review by akinwale — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java (hunk 1 above)
Comment (LuuOW) [reply app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java:412]: Because it's bassiclly what it's causing the keyboard doesn't show properly, another way to fix it it's by adding a hint for the wallet card resource.

Comment (akinwale) [reply app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java:412]: This code is required functionality to change the hint when the input view gains focus, so a better fix would be to explicitly show the soft keyboard when `hasFocus` is true for the corresponding view.

Comment (LuuOW) [reply app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java:412]: uhm... i'll try again then!

Comment (LuuOW) [reply app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java:412]: it's done! i did it without deleting those lines.
